desktop/cmp/input/NumberInput.ts

```diff
@@ -275,6 +275,7 @@ const cmp = hoistCmp.factory<NumberInputModel>(({model, className, ...props}, re
             flex: withDefault(flex, null),
             textAlign: withDefault(props.textAlign, 'right')
         },
+        'data-testid': props.testId,
```

Comment:
Seeing this string in so many places, I wonder if we should make a constant for it, so you could write `[TEST_ID]: props.testId` and not run the risk of a typo